### PR TITLE
Add track opt

### PR DESCRIPTION
**Pull request by h4l**
Add an option to allow loading ga without sending a pageview immediately.

I needed this when dynamically generating a ga loader module in a webpack build. The code already sent pageview commands elsewhere, so didn't need the create/pageview in the (generated) loader module.

### CODE REVIEW

File: dist/isogram-browser.js [hunk 1]
```diff
@@ -78,5 +78,9 @@ window.isogram = function isogram(characters, options) {
     return gaLoader.replace(/\n/g, '') + gaTracker;
   }
 
-  return gaLoader + '\n\n' + gaTracker;
+  if (options.track === undefined || options.track) {
```
Comment (shinnn) [inline dist/isogram-browser.js]: The last `else` is not needed.

```javascript
if (options.track === undefined || options.track) {
  return gaLoader + '\n\n' + gaTracker;
}

return gaLoader;
```

File: index.js [hunk 2]
```diff
@@ -83,5 +83,9 @@ module.exports = function isogram(characters, options) {
     return gaLoader.replace(/\n/g, '') + gaTracker;
   }
 
-  return gaLoader + '\n\n' + gaTracker;
+  if (options.track === undefined || options.track) {
```
Comment (shinnn) [inline index.js]: The last `else` is not needed.

```javascript
if (options.track === undefined || options.track) {
  return gaLoader + '\n\n' + gaTracker;
}

return gaLoader;
```

File: README.md [hunk 3]
```diff
@@ -232,6 +234,13 @@ Default: `false`
 
 Colorize the parameters with [ANSI escape code](https://wikipedia.org/wiki/ANSI_escape_code#Colors).
 
+##### options.track
+
+Type: `Boolean`
+Default: `true`
+
+Include `create` and `pageview` commands after the loading code.
```
Comment (shinnn) [inline README.md]: `false` excludes `create` and `send` commands after the loading code.

File: test-cli.js [hunk 4]
```diff
@@ -56,6 +56,14 @@ test('"isogram" command', function(t) {
     );
   });
 
+  exec(bin + '--no-track', function(err, stdout, stderr) {
+    t.deepEqual(
+      [err, stdout, stderr],
+      [null, isogram({track: false}) + '\n', ''],
+      'should omit create and send commands if using --no-track flag.'
```
Comment (shinnn) [inline test-cli.js]:
```
should omit `create` and `send` commands if using --no-track flag.
```

**Comment by h4l** [comment]
@shinnn Thanks for looking at this, I've updated with your changes.

**Comment by shinnn** [comment]
@h4l OK, merged. Thanks for the pull request with :100: test!

**Comment by h4l** [comment]
@shinnn No problem, thanks for the feedback & merge!

**Comment by shinnn** [comment]
Included in the new version, `v0.6.0`. https://github.com/shinnn/isogram/commit/40dacd87f6a496237082578d372c7809da72a946

**Comment by h4l** [comment]
Fantastic, thanks.

**Pull request merged**